core/string/node_path.cpp

```diff
@@ -331,7 +346,7 @@ NodePath NodePath::simplified() const {
 }
 
 NodePath::NodePath(const Vector<StringName> &p_path, bool p_absolute) {
-	if (p_path.size() == 0) {
+	if (p_path.size() == 0 && !p_absolute) {
```

Comment:
What is the reason for this change and the similar one below? Do I need to include a similar change when making the slice method?

Reply:
The problem was an edge case where `p_absolute` was being ignored if the path was empty.